requirements/test.txt

```diff
@@ -2,3 +2,9 @@ case>=1.3.1
 pytest>=3.0
 pytest-django==3.1.2
 pytz>dev
+
+# Allows freezing of current time in tests
+freezegun==0.3.9
+
+# Provides unittest.mock when running the test suite on Python 2
```

Comment:
That comment seems wrong, the mock package doesn't provide `unittest.mock` but `mock`. `unittest.mock` just happens to be a port to Python 3.x which we could use if we'd only support Python 3.x.